sources/OSG.js

```diff
@@ -1,4 +1,9 @@
 'use strict';
+
+// Check if we need to polyfill
+var polyfill = require( 'osg/polyfill' );
+polyfill.checkSetSupport();
```

Comment:
fine for me but if we add more stuffs to polyfill, maybe we could just say we don't have to call the function but simply needs to import it